WebRoot/homepage/view result.php

```diff
@@ -52,26 +52,30 @@ class="button-filter"
 				<h2><?php echo TEXT[12] ?></h2>
 			</div>
 			<div class="search-result-box">
-				<p>Showing <?php echo $companies['Counter'] ?> results for</p>
+				<p><?php echo TEXT[15] . $companies['Counter'] . TEXT[16] ?></p>
```

Comment:
Use `printf()` with only **one** translation line instead, like with TEXT[1] in `WebRoot/company/text en.php` and `WebRoot/company/view.php`